pyhydroquebec/mqtt_daemon.py

```diff
@@ -120,7 +120,7 @@ def _publish_sensor(self, sensor_type, contract_id,
 
                 # Balance
                 # Publish sensor
-                balance_topic = self._publish_sensor('balance', customer.account_id,
+                balance_topic = self._publish_sensor('balance', customer.contract_id,
```

Comment:
Why did you change that ? This is a breaking change ...

Reply:
**Why you changed the mqtt push data**
This part was modified to only have one topic per account/contract_id. Correct me if I am wrong but the balance topic should be in the contract_id topic.

Reply:
Ok ! Thanks !